airgun/entities/host_new.py

```diff
@@ -951,13 +951,15 @@ def remediate_with_insights(
         view = RemediationView(self.browser)
         view.remediate.click()
 
-    def get_host_facts(self, entity_name):
+    def get_host_facts(self, entity_name, fact=None):
```

Comment:
Doc string?

Reply:
It's pretty straightforward. I don't think docstring is needed.